cool-core/src/main/java/com/nus/cool/core/cohort/refactor/CohortProcessor.java

```diff
@@ -68,6 +77,23 @@ public CohortProcessor(CohortQueryLayout layout) {
     this.result = new CohortRet(layout.getAgetSelectionLayout());
   }
 
+  /**
+   * Read from json file and create a instance of CohortProcessor.
+   *
+   * @param in File
+   * @return instance of file
+   * @throws IOException IOException
+   */
+  public static CohortProcessor readFromJson(File in) throws IOException {
+    ObjectMapper mapper = new ObjectMapper();
+    CohortProcessor instance = mapper.readValue(in, CohortProcessor.class);
+    return instance;
+  }
+
```

Comment:
add the annotation.

  /**
   * Read from json file and create a instance of CohortProcessor.
   *
   * @param path String
   * @return instance of file
   * @throws IOException IOException
   */

Reply:
This function is removed